core/src/network/test_utils.rs

```diff
@@ -196,10 +196,10 @@ pub fn test_wat_always_invalid() -> String {
 
     (func
         (export "__list_functions")
-        (param $allocation i32)
-        (result i32)
+        (param $allocation i64)
+        (result i64)
 
-        (i32.const 0)
+        (i64.const 0)
```

Comment:
what happened here? Why were there still i32s?

Reply:
I don't know, it slipped through cracks with David.  I talked to him about it to confirm.